src/orders/components/OrderRefundDatagrid/datagrid.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  dateCell,
+  moneyCell,
+  readonlyTextCell,
+  tagsCell,
+} from "@dashboard/components/Datagrid/customCells/cells";
+import { AvailableColumn } from "@dashboard/components/Datagrid/types";
+import { OrderDetailsFragment } from "@dashboard/graphql";
+import { getStatusColor } from "@dashboard/misc";
+import { GridCell, Item } from "@glideapps/glide-data-grid";
+import { DefaultTheme } from "@saleor/macaw-ui-next";
+import { IntlShape } from "react-intl";
+
+import { refundGridMessages } from "./messages";
+
+export const orderRefundStaticColumnsAdapter = (
```

Comment:
can't be a hook? creating this as a hook will eliminate one argument (intl) - cleaner : )